HerPublicWebsite/appsettings.Production.json

```diff
@@ -3,16 +3,7 @@
         "BaseUrl": "https://api.epb.digital.communities.gov.uk"
     },
 
-    "GovUkNotify": {

```

Comment:
Do we not need this then? It's for any links back to the site from email we send out. I haven't checked the email contents though.

Reply:
Doesn't look like it - https://www.figma.com/file/4myNK1X0dkDpqNgLTnTeCv/BEIS-Beta-Prototype?type=design&node-id=2719-109598&t=ptY47zy8Dk75hlcI-4

I guess there's not much use going back to the site until we have a way of retrieving/editing a submission.

Reply:
Fair enough, if we don't need it then delete it :)